grunt/config/replace.js

```diff
@@ -37,6 +37,18 @@ module.exports = function (grunt, options) {
         if (configJson._xapi.hasOwnProperty('_activityID') && configJson._xapi._activityID == '') {
           grunt.log.writeln('WARNING: xAPI activityID has not been set');
         }
+
+        // cmi5 is a profile of the xAPI specification and some systems, e.g. SCORM Cloud
+        // do not work well with both manifest types, so remove the one which is not being used.
+        if (configJson._xapi.hasOwnProperty('_specification')) {
```

Comment:
this is horrendously bad practise putting extension specific code in the builder :/ we'll have to conquer this in some other way. perhaps a new task which evals sections of the bower.json and provides an api to that code...

like:

```
[getAllBowerJsons].forEach(function(bowerJson) {
  if (!bowerJson._buildTask) return;
  var execute = new Function("grunt", "path", "configJson", bowerJson._buildTask);
  execute(grunt, path, configJson);
});
```